http/vertx-web-validation/src/test/java/io/quarkus/ts/vertx/web/validation/OpenshiftVertxWebValidationIT.java

```diff
@@ -0,0 +1,7 @@
+package io.quarkus.ts.vertx.web.validation;
+
+import io.quarkus.test.scenarios.OpenShiftScenario;
+
+@OpenShiftScenario
+public class OpenshiftVertxWebValidationIT extends VertxWebValidationIT {
```

Comment:
```suggestion
public class OpenShiftVertxWebValidationIT extends VertxWebValidationIT {
```
CI is failing because of this

Reply:
And also because of this: ` Service "restServiceApp" is invalid: metadata.name: Invalid value: "restServiceApp": a DNS-1035 label must consist of lower case alphanumeric characters or '-'`
Fixed it!